src/index.js

```diff
@@ -992,6 +1001,7 @@ export default class Carousel extends React.Component {
           getSliderStyles(this.props.width, this.props.height),
           this.props.style
         )}
+        dir="ltr"
```

Comment:
where does this `dir` property come in? I don't see it defined anywhere or used anywhere else. if you're trying to use the `rtl` prop then this should be using that prop instead.